lib/linguist/languages.yml

```diff
@@ -7262,6 +7262,7 @@ Visual Basic 6.0:
   type: programming
   color: "#2c6353"
   extensions:
+  - ".bas"
```

Comment:
Is `.bas` the primary extension for Visual Basic 6.0 files? (The first extension is considered the lanuage's primary extension).

Reply:
I would say yes. Like most languages derived from BASIC, the `.bas` is the default extension where you can find standalone code. And that's actually the reason why it took longer to include, because it's the same extension as VBA and I had to make sure that good heuristics were put in place to make this work.